src/containers/edit-profile/notification-tab/NotificationTab.styles.ts

```diff
@@ -1,9 +1,16 @@
 import { TypographyVariantEnum } from '~/types'
-import { rootContainer } from '~/containers/edit-profile/common.style'
+import {
+  innerContainer,
+  rootContainer,
+  updateProfileBtn
+} from '~/containers/edit-profile/common.style'
 import { switchStyle } from '~/containers/my-quizzes/quiz-settings-container/QuizSettingsContainer.styles'
 
 export const styles = {
   root: rootContainer,
+  notificationInnerContainer: {
+    ...innerContainer
+  },
```

Comment:
Use just `notificationInnerContainer: innerContainer` because there is no need to clone object with spread operator